# Robot Parameters
wheel_base: 0.1                             # The distance between the wheels
wheel_radius: 0.26                          # The radius of the wheels

# Odometer Parameters
odom_frame_id: "odom"                       # The name of the odometry tf frame
body_frame_id: "base_footprint"                  # The name of the body tf frame
left_wheel_joint: "wheel_left_joint"         # The name of the left wheel joint
right_wheel_joint: "wheel_right_joint"       # The name of the right wheel joint
